cirq-core/cirq/experiments/qubit_characterizations.py

```diff
@@ -54,11 +85,11 @@ class Cliffords:
         s1_y
     """
 
-    c1_in_xy: list[list[ops.SingleQubitCliffordGate]]
-    c1_in_xz: list[list[ops.SingleQubitCliffordGate]]
-    s1: list[list[ops.SingleQubitCliffordGate]]
-    s1_x: list[list[ops.SingleQubitCliffordGate]]
-    s1_y: list[list[ops.SingleQubitCliffordGate]]
+    c1_in_xy: _CliffordGateSequence = attrs.field(converter=_CliffordGateSequence)
+    c1_in_xz: _CliffordGateSequence = attrs.field(converter=_CliffordGateSequence)
+    s1: _CliffordGateSequence = attrs.field(converter=_CliffordGateSequence)
+    s1_x: _CliffordGateSequence = attrs.field(converter=_CliffordGateSequence)
+    s1_y: _CliffordGateSequence = attrs.field(converter=_CliffordGateSequence)
```

Comment:
If possible, we should avoid the `_CliffordGateSequence` wrapper.
How about just keeping the original list type and using `converter=_canonize_clifford_sequences`?

Current unit test expands all fields so there is no time saved on their delayed conversion through `_CliffordGateSequence`.

Reply:
that was the first version of the code, but I needed to add the class to keep the tests the explicitly check the content of the lists ... that is some tests check that the clifford are written in terms of X/Z or X/Y

Reply:
Then perhaps we can add some `canonized: bool` flag for the `_single_qubit_cliffords` factory which will return old Cliffords for the X/Z, X/Y test and canonized elsewhere.  Would that work?